tests/test_casper.py

```diff
@@ -52,32 +53,58 @@ def proposed(connection, deployed):
     return casper.propose(connection)
 
 
+@pytest.fixture
+def block_hash(connection):
+    return casper.get_blocks(connection, depth=1).pop().get('blockHash')
+
+
+def test_get_blocks(connection, block_fields):
+    blocks = casper.get_blocks(connection, depth=100)
+    assert isinstance(blocks, list)
+    assert len(blocks) > 0
+    for block in blocks:
+        for field in block.keys():
+            assert field in block_fields
+
+
+def test_get_block(connection, block_hash, block_fields):
+    block = casper.get_block(connection, block_hash)
+    assert isinstance(block, dict)
+    assert len(block) > 0
+    for field in block.keys():
+        assert field in block_fields
+
+
 def test_deploy(deployed):
     assert deployed == {'message': 'Success!', 'success': True}
 
 
 def test_propose(proposed):
     assert isinstance(proposed, dict)
-    assert proposed['success'] == True
+    assert proposed['success']
```

Comment:
`==` is not needed but i think we should stay with `is True`. what if`proposed['success'] == 'false'` this test will pass but it should not

Reply:
ok, I agree with `is True`

Reply:
done